sale_stock_order_line_quantity_readonly/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module extends the functionality of the Sales module to restrict users from updating the quantity of products in sale order lines:
+
+- When the sale order is locked/confirmed, users are still able to adjust the quantities.
```

Comment:
This is not the standard. You should not write this or mention the module that change that behavior.

Reply:
And you can test this on Runbot ;)